File: 944. Delete Columns to Make Sorted/n_square.cpp
class Solution {
    public:
        int minDeletionSize(vector<string>& strs){
            int del_cols = 0;
            for (int col=0; col < strs[0].length(); col++)
                for (int row=0; row < strs.size() - 1; row++){
                    if (strs[row][col] > strs[row+1][col]){
                        del_cols += 1;
                        break;
                    }
                }
            return del_cols;
        }
};

File: 944. Delete Columns to Make Sorted/n_square.py
class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        del_cols = 0
        for col in range(len(strs[0])):
            for row in range(len(strs) - 1):
                if strs[row + 1][col] < strs[row][col]:
                    del_cols += 1
                    break
        return del_cols